rskj-core/src/main/java/co/rsk/peg/whitelist/constants/WhitelistConstants.java

```diff
@@ -1,11 +1,16 @@
 package co.rsk.peg.whitelist.constants;
 
+import co.rsk.bitcoinj.core.NetworkParameters;
 import co.rsk.peg.vote.AddressBasedAuthorizer;
 
 public class WhitelistConstants {
-
+    protected NetworkParameters btcParams;
```

Comment:
Is it necessary to include `btcParams` as part of the Whitelist objects?  I noticed this attribute is already in BridgeConstants, and putting it in other places could affect the data modeling. If we see that data model to get a good normalization we have to avoid spreading the same information/attribute in different objects.

Reply:
Makes life easier :) I can get the network params from the constants instead of passing the NetowrkParam object around.

I agree that it adds some duplication, but the value is also tied to the class name (MainnetConstants => MainnetParams). I'm open for discussing it, there's room for improvement in many thing s we've done so far definitely